mixins/localize/README.md

```diff
@@ -121,6 +121,66 @@ If your localized string contains arguments, pass them as a key-value object as
 
 ```javascript
 render() {
-  return html`<p>${this.localize('hello', {firstName: 'Mary'})}</p>`;
+  return html`<p>${this.localize('hello', { firstName: 'Mary' })}</p>`;
 }
 ```
+
+## `localizeHTML()`
```

Comment:
I've updated this to reflect the new state of things. I purposefully didn't document advanced usage where consumers could pass anything they want... I figured we could keep that to ourselves for now in the hopes that we could create more helpers for commonly requested things.